IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetry.h

```diff
@@ -26,11 +26,31 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/*
+ •    InteractiveNoCacheLookupInvolved = 0, request goes to ESTS for interactive call for which there is no cache look-up involved (N/A for S2S).
+ •    ForcedTokenRefresh = 1, request goes to ESTS because caller requested to forcefully refresh the cache.
+ •    CacheMiss = 2, request goes to ESTS because cache entry for the requested token does NOT exist.
+ •    CachedTokenExpired = 3, request goes to ESTS because cache entry for the requested token does exist but token has expired.
+ •    ProactiveTokenRefresh = 4, request goes to ESTS because refresh_in was used and existing non-expired token needs to be refreshed proactively.
+ •    CachingMechanismNotImplemented = 5, request goes to ESTS because client (for Non-MSAL client specifically for now) has not implemented any caching mechanism.
+ •    BLANK, if client is not aware of the LLT policy and its telemetry update and doesn’t update their code to send us this telemetry signal yet.
+ */
+
+typedef NS_ENUM(NSInteger, TokenCacheRefresh)
+{
+    NoCacheLookupInvolved,
+    ForceRefresh,
+    NoCachedAT,
+    ExpiredAT,
+    RefreshExpiredAT,
+    CachingMechanismNotImplemented,
+};
+
 @interface MSIDCurrentRequestTelemetry : NSObject <MSIDTelemetryStringSerializable>
 
 @property (nonatomic) NSInteger schemaVersion;
 @property (nonatomic) NSInteger apiId;
-@property (nonatomic) BOOL forceRefresh;
+@property (nonatomic) TokenCacheRefresh tokenCacheRefresh;
```

Comment:
I don't see ForceRefresh being set in the silent token request. Can you double check if there's a missing update?

Reply:
ForceRefresh is set at MSALPublicCleintApplication. I have a PR for msal repo to update that change